.github/workflows/docker-publish.yml

```diff
@@ -0,0 +1,77 @@
+name: Docker
+
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+on:
+  push:
+    branches: ["main"]
```

Comment:
a quick way to test is to add feat/docker-ci here and remove it before merging to main :)

Reply:
got it - tested!